packages/sdks/snippets/gen1-react/src/routes/targeted-page/is-logged-in-request.tsx

```diff
@@ -0,0 +1,9 @@
+import { builder } from '@builder.io/react';
+
+export default async ({ model, urlPath }: { model: string; urlPath: string }) =>
```

Comment:
- unnecessary async
- for consistency all requests fns should have the same name. I see some don't have a name at all, and some are called `request`. I find `request` to be a bit of a special keyword in JavaScript land. I'd opt for a name like `targettedRequest`

Reply:
Thank you, @samijaber. I've updated the code to remove the last few `async` keywords. I've also used the `GetContentOptions` type in Gen 1 now that I know where it is.